python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -65,6 +66,9 @@ def __init__(self) -> None:
         # imSim Component
         self.imsim_cmpt = None
 
+        # Maximum Noll index
+        self.max_noll_index = 22
```

Comment:
Well, the max_noll_index is used in different functions and I thought it was cleaner for it to be a class variable instead of having to pass it down as an attribute through multiple nested functions

Reply:
Ok. But let's set it to `None` here like the rest so that the default value will come from the command line arguments. How's that sound?